modules/display.c

```diff
@@ -365,6 +367,37 @@ static gboolean backlight_ioctl(int value)
 	return status;
 }
 
+/* Set DPMS mode for X11 display to on or off
+ *
+ * @params dpms_on To force the mode into on or off (blanked) mode.
+ */
+static void display_dpms(gboolean dpms_on)
+{
+	Display* dpy = NULL;
+
+	dpy = XOpenDisplay(NULL);
+	if (dpy == NULL) {
+		dpy = XOpenDisplay(":0.0");
+	}
+
+	if (dpy == NULL) {
+		mce_log(LL_CRIT, "display_dpms: unable to open display");
+		return;
+	}
+
+	if (dpms_on) {
+		DPMSEnable(dpy);
+		DPMSForceLevel(dpy, DPMSModeOn);
+	} else {
+		DPMSEnable(dpy);
+		usleep(100000);
+		DPMSForceLevel(dpy, DPMSModeOff);
+	}
```

Comment:
I'd add `if (DPMSCapable(dpy))` around this code

Reply:
Will do